FACG6/city-in-the-south
# POST:Add new member

### Related:
related to #35 

- query ``` checkUsername ``` for  check if the username exists or not #93 
- query  `checkEmail ` for check if the email  exists or not 
- query  ` addMmeber `  for add new member 
- integration test for the route with its query.

### Summary:
how to test this PR: 
 -  ```git fetch origin  add-new-member ```
 -  ```git checkout   add-new-member ```
 -  ```git pull origin   add-new-member```
 - ```npm run server```
 - on postman run this route ``` `/api/v1/members  ``` with method ` POST ` 
 and body    :
```javascript
{
   "username":"fatma",
   "email":"fatma@gmail.com",
    "pass":"123456789"
}
```
_**Hint**_: the pass should be at leat 8 characters 

### Checklist Criteria
- [x] **Branch is up to date with master**
- [x] **The project runs**
- - [x] You can download and install locally, and run the project locally
- - [x] All existing and new features run without bugs
- - [x] The build script runs without error
- [ ] **Code is consistent**
- - [x] Linter is used, no error messages
- - [x] Code is consistent with agreed team style
- - [x] Naming of files is consistent with project (e.g. all files named with camelCase)
- - [x] Console.log is only used for crucial system messages (e.g. which port server is on).
- - [x] Any console.logs used for debugging have been removed.
